### ISSUE TITLE
Hard to follow and understand

### ISSUE THREAD

**Issue by VeraKulemina** [learn-co-curriculum/phase-1-lecture-videos-map-part-2#4]
### Canvas Link

https://learning.flatironschool.com/courses/5234/pages/video-map-part-2?module_item_id=478964

### Concern

Very hard to understand what the lecturer is doing in this video. I have mush information because she deletes and renews all her code and not just comments it out for example. Also very confusing what is going on there because the terminology is new for me.

**Comment by lizbur10** [comment]
Hi again @VeraKulemina ! I have made note of this one as well, and it will also be addressed the next time this video is updated. 💙
